ux/src/main/java/pro/streem/ar/sceneform/ux/BaseArFragment.java

```diff
@@ -514,7 +529,9 @@ protected void onWindowFocusChanged(boolean hasFocus) {
     @Override
     public void onPause() {
         super.onPause();
-        stop();
+        if (!supportsPictureInPicture) {
+            stop();
```

Comment:
should there be a call to `stop()` in `onStop()` when `supportsPictureInPicture == true`?

Reply:
`stop()` gets called in `onDestroy()` which is called shortly after `onStop()`, but I think you are right that for consistency it makes sense to also add a call in `onStop()`. There's no harm in calling it twice, because the call in `onDestroy()` will not do anything if already called. The call in `onDestroy()` seems redundant and probably is, since it should always be called in either `onPause()` or `onStop()` (once I implement your suggestion), but I hesitate to remove it just in case the wise folks at Google put it there for a good reason.

Thanks!

Reply:
Updated.